### PR TITLE
[FEATURE] (#679) i18n for editor plugins

### PR DESCRIPTION
**Pull request by hyukmin-kwon**
[DESC.] i18n for editor plugins
* plugins/webida.editor.text-editor
* plugins/webida.editor.code-editor
* plugins/webida.editor.code-editor.content-assist.tern

### CODE REVIEW

File: apps/ide/src/plugins/webida.editor.code-editor/CodeEditorContextMenu.js [hunk 1]
```diff
@@ -90,7 +92,7 @@ define([
                     sourceItems['Comment Out Selection'] = allItems.editMenuItems['&Source']['Comment Out Selection'];
                 }
                 // Code Folding
-                sourceItems['&Fold'] = allItems.editMenuItems['&Source']['&Fold'];
+                sourceItems['&Fold'] = allItems.editMenuItems['&Source']['&Fold'];
```
Comment (hwshim) [inline apps/ide/src/plugins/webida.editor.code-editor/CodeEditorContextMenu.js]: Please remove white spaces.

File: apps/ide/src/plugins/webida.editor.code-editor/CodeEditorContextMenu.js [hunk 2]
```diff
@@ -99,19 +101,40 @@ define([
                     }
                     sourceItems['B&eautify All'] = allItems.editMenuItems['&Source']['B&eautify All'];
                 }
-                items['So&urce'] = sourceItems;
+                items['So&urce'] = sourceItems;
```
Comment (hwshim) [inline apps/ide/src/plugins/webida.editor.code-editor/CodeEditorContextMenu.js]: Please remove white spaces.

File: apps/ide/src/plugins/webida.editor.code-editor/CodeEditorContextMenu.js [hunk 3]
```diff
@@ -99,19 +101,40 @@ define([
                     }
                     sourceItems['B&eautify All'] = allItems.editMenuItems['&Source']['B&eautify All'];
                 }
-                items['So&urce'] = sourceItems;
+                items['So&urce'] = sourceItems;               
 
                 // Go to
-                items['&Go to Definition'] = allItems.navMenuItems['&Go to Definition'];
+                items['&Go to Definition'] = allItems.navMenuItems['&Go to Definition'];
```
Comment (hwshim) [inline apps/ide/src/plugins/webida.editor.code-editor/CodeEditorContextMenu.js]: Please remove white spaces.

File: apps/ide/src/plugins/webida.editor.code-editor/CodeEditorContextMenu.js [hunk 4]
```diff
@@ -99,19 +101,40 @@ define([
...
+                items['So&urce'] = sourceItems;               
 
                 // Go to
-                items['&Go to Definition'] = allItems.navMenuItems['&Go to Definition'];
+                items['&Go to Definition'] = allItems.navMenuItems['&Go to Definition'];                
 
                 if (viewer.isDefaultKeyMap()) {
                     items['G&o to Line'] = allItems.navMenuItems['G&o to Line'];
-                }
+                }
```
Comment (hwshim) [inline apps/ide/src/plugins/webida.editor.code-editor/CodeEditorContextMenu.js]: Please remove white spaces.

File: apps/ide/src/plugins/webida.editor.code-editor/CodeEditorViewer.js [hunk 5]
```diff
@@ -1329,26 +1331,26 @@ define([
                 var history = editor.getHistory();
                 if (history) {
                     if (history.done && history.done.length > 0) {
-                        items['&Undo'] = menuItems.editMenuItems['&Undo'];
+                        items['&Undo'] = menuItems.editMenuItems['&Undo'];
```
Comment (hwshim) [inline apps/ide/src/plugins/webida.editor.code-editor/CodeEditorViewer.js]: Please remove white spaces.

File: apps/ide/src/plugins/webida.editor.code-editor/CodeEditorViewer.js [hunk 6]
```diff
@@ -1329,26 +1331,26 @@ define([
...
                     if (history.undone && history.undone.length > 0) {
-                        items['&Redo'] = menuItems.editMenuItems['&Redo'];
+                        items['&Redo'] = menuItems.editMenuItems['&Redo'];                        
                     }
                 }
 
                 // Delete
                 items['&Delete'] = menuItems.editMenuItems['&Delete'];
-
+
```
Comment (hwshim) [inline apps/ide/src/plugins/webida.editor.code-editor/CodeEditorViewer.js]: Please remove white spaces.

File: apps/ide/src/plugins/webida.editor.code-editor/CodeEditorViewer.js [hunk 7]
```diff
@@ -1329,26 +1331,26 @@ define([
...
                 }
 
                 // Delete
                 items['&Delete'] = menuItems.editMenuItems['&Delete'];
-
+                
                 // Select All, Select Line
                 items['Select &All'] = menuItems.editMenuItems['Select &All'];
-                items['Select L&ine'] = menuItems.editMenuItems['Select L&ine'];
+                items['Select L&ine'] = menuItems.editMenuItems['Select L&ine'];
```
Comment (hwshim) [inline apps/ide/src/plugins/webida.editor.code-editor/CodeEditorViewer.js]: Please remove white spaces.

File: apps/ide/src/plugins/webida.editor.code-editor/CodeEditorViewer.js [hunk 8]
```diff
@@ -1329,26 +1331,26 @@ define([
...
-                items['Select L&ine'] = menuItems.editMenuItems['Select L&ine'];
+                items['Select L&ine'] = menuItems.editMenuItems['Select L&ine'];                
 
                 // Line
                 var lineItems = {};
 
                 // Line - Move Line Up, Move Line Down, Copy, Delete
                 lineItems['&Indent'] = menuItems.editMenuItems['&Line']['&Indent'];
-                lineItems['&Dedent'] = menuItems.editMenuItems['&Line']['&Dedent'];
+                lineItems['&Dedent'] = menuItems.editMenuItems['&Line']['&Dedent'];
```
Comment (hwshim) [inline apps/ide/src/plugins/webida.editor.code-editor/CodeEditorViewer.js]: Please remove white spaces.

**Comment by hyukmin-kwon** [comment]
* Unintended white spaces are removed.

**Pull request merged**